### PR TITLE
Refactor/Move to GitHub/New ETL Tables

### PR DESCRIPTION
**Pull request by dsteinich**

### CODE REVIEW

File: JenkinsFile [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+    stage('Clean Workspace') {
+      steps {
+        cleanWs()
+      }
+    }
+    stage('Git Clone') {
+      steps {
+        checkout([$class: 'GitSCM', branches: [[name: '*/master']],
+        doGenerateSubmoduleConfigurations: false,
+        extensions: [],
```
Comment (ssoper-usgs) [inline JenkinsFile]: Are lines 17-18 needed?

File: JenkinsFile [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+        submoduleCfg: [],
+        userRemoteConfigs: [[credentialsId: 'CIDA-Jenkins-GitHub',
+        url: 'https://github.com/usgs/aqts-capture-db.git']]])
+      }
+    }
+    stage('Download liquibase jar') {
+      steps {
+        sh '''mkdir $WORKSPACE/rc
+          if [ ! -f rc/liquibase ]; then
+            /usr/local/bin/aws s3 cp s3://owi-common-resources/resources/InstallFiles/liquibase/liquibase-$LIQUIBASE_VERSION.tar.gz $WORKSPACE/rc/liquibase.tar.gz
```
Comment (ssoper-usgs) [inline JenkinsFile]: Thinking out loud: we specify liquibase version 3.8.5 as part of the travis build - is there a good way to make sure we're using that same version in this liquibase script?

File: JenkinsFile [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+            /usr/bin/tar xzf $WORKSPACE/rc/liquibase.tar.gz --overwrite -C $WORKSPACE/rc
+            /usr/local/bin/aws s3 cp s3://owi-common-resources/resources/InstallFiles/postgres/$JDBC_JAR $WORKSPACE/rc/lib/$JDBC_JAR
+          fi
+        '''
+      }
+    }
+    stage('Run liquibase') {
+      steps {
+        script {
+          def secretsString = sh(script: '/usr/local/bin/aws ssm get-parameter --name "/aws/reference/secretsmanager/NWCAPTURE-DB-TEST" --query "Parameter.Value" --with-decryption --output text --region "us-west-2"', returnStdout: true).trim()
```
Comment (ssoper-usgs) [inline JenkinsFile:33]: This is neat - these all coming from secrets/param store in aws now?  Do any credentials still need to exist in Jenkins?

File: JenkinsFile (hunk 2 above)
Comment (ayan-usgs) [reply JenkinsFile]: We could in principle. We'd have to drop the new liquibase in S3.

File: JenkinsFile (hunk 3 above)
Comment (ayan-usgs) [reply JenkinsFile:33]: This is great!

Comment (dsteinich) [reply JenkinsFile:33]: Correct, all from AWS, none from Jenkins

File: JenkinsFile (hunk 2 above)
Comment (dsteinich) [reply JenkinsFile]: We should get errors If the version used is not compatible with the script.. Also limited by what has been put on S3...

File: docker-compose.yml [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+    build:
+      context: .
+    networks:
+      aqts_capture:
+        ipv4_address: ${DB_IPV4}
+    environment:
+      - POSTGRES_PASSWORD=${POSTGRES_PASSWORD}
+    ports:
+      - ${DB_PORT}:5432
+    container_name: ${AQTS_DATABASE_ADDRESS}
```
Comment (ssoper-usgs) [inline docker-compose.yml:36]: Is the container_name intended to be the db address here?

File: liquibase/Dockerfile [hunk 5]
```diff
@@ -0,0 +1,26 @@
...
+############################################
+# Install Liquibase
+############################################
+
+ARG LIQUIBASE_VERSION
+ARG A_JDBC_JAR
+ENV JDBC_JAR=$A_JDBC_JAR
+ENV LIQUIBASE_HOME $HOME
+ENV LIQUIBASE_WORKSPACE $LIQUIBASE_HOME/workspace
+ENV LOCALONLY "-c listen_addresses='127.0.0.1, ::1'"
```
Comment (ssoper-usgs) [inline liquibase/Dockerfile]: Can LOCALONLY be LOCAL_ONLY instead?

File: JenkinsFile (hunk 1 above)
Comment (dsteinich) [reply JenkinsFile]: Probably not - cruft from current repository - will remove...

File: liquibase/changeLogs/capture/tables/timeSeriesGapTolerances/timeSeriesGapTolerances.sql [hunk 6]
```diff
@@ -0,0 +1,9 @@
+create table if not exists ${AQTS_SCHEMA_NAME}.time_series_gap_tolerances
+(time_series_gap_tolerances_id           bigint generated by default as identity (start with 1)
+,time_series_unique_id                   character varying (32)
+,start_time                              timestamp with time zone
+,end_time                                timestamp with time zone
+,tolerance_in_minutes                    integer
+,primary key (time_series_gap_tolerances_id)
```
Comment (ssoper-usgs) [inline liquibase/changeLogs/capture/tables/timeSeriesGapTolerances/timeSeriesGapTolerances.sql:7]: On lines 2 and 7, can we use tolerance vs. tolerances? EDIT: Wait, this is a pattern -> please disregard my comment.

File: docker-compose.yml (hunk 4 above)
Comment (dsteinich) [reply docker-compose.yml:36]: yes - you could argue the parameter is serving two purposes - a server name when using the docker network (like the travis build) or an IP address when not (some local runs) 
I think it is less confusing to have them be same.

File: liquibase/Dockerfile (hunk 5 above)
Comment (dsteinich) [reply liquibase/Dockerfile]: Sure

File: liquibase/scripts/dbInit/z1_postgres_liquibase.sh [hunk 7]
```diff
@@ -0,0 +1,36 @@
...
+	--liquibaseCatalogName=public \
+	--liquibaseSchemaName=public \
+	update \
+	-DAQTS_DATABASE_NAME=${AQTS_DATABASE_NAME} \
+	-DAQTS_DB_OWNER_USERNAME=${AQTS_DB_OWNER_USERNAME} \
+	-DAQTS_DB_OWNER_PASSWORD=${AQTS_DB_OWNER_PASSWORD} \
+	-DAQTS_SCHEMA_OWNER_USERNAME=${AQTS_SCHEMA_OWNER_USERNAME} \
+	-DAQTS_SCHEMA_OWNER_PASSWORD=${AQTS_SCHEMA_OWNER_PASSWORD}
+
+# postgres to capture capture db table scripts
```
Comment (ssoper-usgs) [inline liquibase/scripts/dbInit/z1_postgres_liquibase.sh]: Might have an extra "capture" here.

**Review by ssoper-usgs — APPROVED** [review]

File: JenkinsFile (hunk 2 above)
Comment (ssoper-usgs) [reply JenkinsFile]: Right, that makes sense.

**Review by ayan-usgs — APPROVED** [review]

**Pull request merged**